pkg/cmd/server/api/v1/types.go

```diff
@@ -377,6 +377,9 @@ type AssetConfig struct {
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL"`
 
+	// CLIDownloadPublicURLs holds links to download client tools, where key is the platform name and value is the download URL (optional)
```

Comment:
we also don't expose doc links in config... I would expect doc links and cli download links to use the same mechanism